### PR TITLE
Update Spark Roadmap

### PR DESCRIPTION
**Pull request by rgbkrk**
Just a few quick updates to the spark roadmap.

### CODE REVIEW

File: spark.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ Approaches:
 - https://github.com/mozilla/jupyter-spark
 - [Progress monitor](https://github.com/jupyter/jupyter_client/issues/209#issuecomment-260789525)
 
+Progress:
+
+* [ProgressBar](https://github.com/ipython/ipython/blob/345d51130aec9644dbeb08c3e54d93f83babbf3f/IPython/core/display.py#L732) is now a builtin on the IPython library
```
Comment (blink1073) [inline spark.md]: How about "is now available in the IPython library"?

Comment (rgbkrk) [reply spark.md]: Good point, it's definitely not a builtin. Silly me.

**Comment by rgbkrk** [comment]
Thanks for the feedback @blink1073 -- updated!

**Comment by blink1073** [comment]
👍

**Pull request merged**